features/step_definitions/commit_steps.rb

```diff
@@ -12,18 +12,18 @@
 
 Then(/^(?:now )?(I|my coworker) (?:still )?(?:have|has) the following commits$/) do |who, commits_table|
   user = (who == 'I') ? :developer : :coworker
-  commits_table.map_headers!(&:downcase)
   in_repository user do
-    verify_commits commits_table.hashes
+    verify_commits commits_table
   end
 end
 
 
 Then(/^I am left with my original commits$/) do
-  verify_commits @initial_commits_table.hashes
+  @initial_commits_table.map_headers!(&:upcase)
```

Comment:
This hack will go away once I clean up https://github.com/Originate/git-town/blob/master/features/step_definitions/commit_steps.rb#L3 separately. I wanted to keep this PR lean.